sdk/core/core-http/CHANGELOG.md

```diff
@@ -1,10 +1,11 @@
 # Release History
 
-## 1.1.4 (2020-06-30)
+## 1.1.4 (2020-07-02)
 
 - Fix issue with flattened model serialization, where constant properties are being dropped [PR #8658](https://github.com/Azure/azure-sdk-for-js/pull/8658)
 - Switch to use `x-ms-useragent` header key for passing user agent info in browsers [PR #9490](https://github.com/Azure/azure-sdk-for-js/pull/9490)
 - Refactor `ExponentialRetryPolicy` and `SystemErrorRetryPolicy` to share common code [PR #9667](https://github.com/Azure/azure-sdk-for-js/pull/9490)
+- Fix a regression in ExponentialRetryPolicy where we retried when response is undefined [PR #9852](https://github.com/Azure/azure-sdk-for-js/pull/9852)
```

Comment:
We don't need a changelog entry for this was a bug introduced in 1.1.4 not 1.1.3. So, from a user's perspective this does not affect them

Reply:
Removed this entry